app/models/User.coffee

```diff
@@ -250,6 +250,29 @@ module.exports = class User extends CocoModel
         @trigger 'email-verify-error'
     })
 
+  sendKeepMeUpdatedVerificationCode: (code) ->
+    $.ajax({
+      method: 'POST'
+      url: "/db/user/#{@id}/keep-me-updated/#{code}"
+      success: (attributes) =>
+        this.set attributes
```

Comment:
`this.set` should be `@set` in CoffeeScript (and on line 269)

Reply:
this is how our current verify email page works, necessary to change?